src/mono/wasm/Wasm.Build.Tests/BuildTestBase.cs

```diff
@@ -198,6 +198,8 @@ public BuildTestBase(ProjectProviderBase providerBase, ITestOutputHelper output,
             return (res, logFilePath);
         }
 
+        protected bool IsDotnetWasmFromRuntimePack(BuildArgs buildArgs) => !(buildArgs.AOT || buildArgs.Config == "Release");
```

Comment:
The second part (config == release) is correct only when workload is installed. If we would run test in release build with dotnet-none is wouldn't be correct

Reply:
Thanks for spotting it, adding `IsUsingWorkloads` check.